hub/executors/indexers/vector/annoy/annoy.yml

```
@@ -0,0 +1,8 @@
+!AnnoyIndexer
+metas:
+  workspace: $TMP_WORKSPACE
```

For indexer, it's better to use the `--volumes` to amount the volume. Here is an example in the faiss indexer. [https://github.com/jina-ai/jina-hub/tree/master/hub/executors/indexers/vector/faiss#usages](https://github.com/jina-ai/jina-hub/tree/master/hub/executors/indexers/vector/faiss#usages)
```suggestion
  workspace: ./workspace
```